server/handlers/find.js

```diff
@@ -43,6 +43,9 @@ export const findById = async _id => {
   return user;
 };
 
+/** Returns a user that matches handle
+ * @param {string} handle
+ */
```

Comment:
Todo: these should probably explicitly @return {UserData} or whatever your user data object that's being returned here is.

Reply:
fixed